sklearn/utils/testing.py

```diff
@@ -877,10 +877,11 @@ def assert_run_python_script(source_code, timeout=60):
         cmd = [sys.executable, source_file]
         cwd = op.normpath(op.join(op.dirname(sklearn.__file__), '..'))
         env = os.environ.copy()
+        env["PYTHONPATH"] = cwd
```

Comment:
I think it's possible that we need to retain any existing PYTHONPATH entries to be able to import sklearn and its dependencies.